### DISCUSSION TITLE
What `<UnoSingleProject>true</UnoSingleProject>` does for library projects?

### DISCUSSION THREAD

**Discussion by xperiandri** [unoplatform/uno#16526] [Q&A]
I wonder what `<UnoSingleProject>true</UnoSingleProject>` does for library projects.
As before using it if I specify only `net8.0;net8.0-windows10.0.22xxx` as target frameworks Android and iOS worked well.
But after migrating to a single project and using this parameter iOS and Android no longer work

**Comment by agneszitte** [comment]
cc @jeromelaban, @dansiegel

**Reply by jeromelaban** [reply]
`<UnoSingleProject>true</UnoSingleProject>` defines all sorts of implicit behaviors, such as default globbings. It is not expected that `net8.0;net8.0-windows10.0.22xxx` can work on iOS/Android, even before 5.2, because the inheritance for UI Element is incorrect on net8.0 when compared to net8.0-ios/android. You need to specify `net8.0-ios` and `net8.0-android` for controls to work.
